test/Microsoft.Extensions.Internal.Test/project.json

```diff
@@ -1,7 +1,6 @@
 {
   "buildOptions": {
-    "warningsAsErrors": true,
-    "debugType": "full"
```

Comment:
Forcing it to build `full` causes it to fail building on xplat. However, it does generate full pdbs on Windows, so we have some coverage here.